dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/UserGroupPopulator.java

```diff
@@ -0,0 +1,51 @@
+package org.hisp.dhis.user;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hisp.dhis.common.IdentifiableObjectStore;
+import org.hisp.dhis.security.acl.AclService;
+import org.hisp.dhis.system.startup.TransactionContextStartupRoutine;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Slf4j
+public class UserGroupPopulator
```

Comment:
Is this class needed? seems the only function is commented out

Reply:
removed